frontend/redux/UserReducer.tsx

```diff
@@ -0,0 +1,22 @@
+import { combineReducers } from "redux";
+
+const INITIAL_STATE = {
+  _id: "",
+  name: "",
+  email: "",
+  token: "",
+  superUser: false,
```

Comment:
Would be ideal to use the `Admin` object (which really should have a `superUser`). We also don't need to store the id since it doesn't actually see any use for our application.